web/src/layouts/SidebarLayout/SidebarLayout.tsx

```diff
@@ -13,6 +13,42 @@ type SidebarLayoutProps = {
 const SidebarLayout = ({ children }: SidebarLayoutProps) => {
   const [sidebarOpen, setSidebarOpen] = useState(false)
 
+  useEffect(() => {
+    const script = document.createElement('script')
+    script.innerHTML = `
+    window.CommunitySearch = Object.assign({}, window.CommunitySearch, {
+      baseUrl: "https://search.orbit.love/widget/f6f0c448-cc87-432c-a0c5-a208c24afb18",
+      options: {
+        showCornerIcon: false,
+        searchBoxContainer: "#community-search-box",
+        searchBoxPlaceholder: "Community search...",
+      },
+    });
+
+    if (window.CommunitySearch.init) {
+      window.CommunitySearch.init();
+    }
+  `
+    document.body.appendChild(script)
+    document.body.appendChild(script)
+
+    return () => {
+      document.body.removeChild(script)
+    }
+  }, [])
+
+  useEffect(() => {
```

Comment:
This looks a little bit weird. Does orbit have any docs about how to best integrate this into a React page?